packages/mirrorful/editor/src/components/Onboarding/pages/NamePrimary.tsx

```diff
@@ -113,12 +113,21 @@ export function NamePrimary({
           }}
           bgColor={primaryColor}
         />
-        <Text
-          css={{ alignSelf: 'flex-start', marginBottom: '4px' }}
-          fontWeight="bold"
-        >
-          Color Name:
-        </Text>
+        <Flex>
+            <Text
+              css={{ alignSelf: 'flex-start', marginBottom: '4px' }}
+              fontWeight="bold"
+            >
+              Color Name:
+            </Text>
+            <Tooltip
+                  placement="right"
+                  closeDelay={500}
+                  hasArrow
+                  label={"Variable names don't need a hyphen."}>
+                <InfoIcon css={{ marginTop: '5px', marginLeft: '4px' }} />
+            </Tooltip>
+          </Flex>
```

Comment:
I think this is not needed here, it's too much on this page. I think also because the word is "Color Name" vs "Variable Name" its more clear. 

<img width="503" alt="Screen Shot 2023-03-14 at 9 47 31 AM" src="https://user-images.githubusercontent.com/29822597/225077623-7ef7abc3-1c99-4894-8d37-a06b810bcd9d.png">